mobile/android/android-device.ts

```diff
@@ -173,6 +173,10 @@ export class AndroidDevice implements Mobile.IAndroidDevice {
 		return (() => {
 			var startPackageCommand = this.composeCommand("shell am start -a android.intent.action.MAIN -n %s/%s", packageName, this.$staticConfig.START_PACKAGE_ACTIVITY_NAME);
 			this.$childProcess.exec(startPackageCommand).wait();
+
+			if (options.logging) {
```

Comment:
Do you think that it is possible for the users to confuse ```--log``` and ```--logging```? If so, can we find a better name for one of the options? For example, the user must write
```$ tns deploy ios --logging --log trace```